content/_data/changelogs/weekly.yml

```diff
@@ -7957,6 +7957,97 @@
   # pull: 4777 (PR title: Use the 'agent' term more often)
   # pull: 4778 (PR title: Fixed some style errors in less files to reduce output during build)
 
+- version: "2.242"
+  date: 2020-06-23
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4802
+    issue: 62622
+    authors:
+    - daniel-beck
+    message: |-
+      * Do not log exceptions when browsing to the plugin manager while private source plugins are installed (regression in 2.240).
+  - type: bug
+    category: bug
+    pull: 4813
+    issue: 61438
+    authors:
+    - rahulsom
+    references:
+      - issue: 61438
+      - url: https://github.com/stapler/stapler/releases/tag/stapler-parent-1.260
+        title: Stapler 1.260 changelog
+    message: |-
+      Fix <code>IllegalArgumentException</code> for Choice Parameter when saving job configuration.
+  - type: bug
+    category: bug
+    pull: 4813
+    issue: 61438
+    authors:
+    - rahulsom
+    references:
+      - pull: 4813
+      - url: https://github.com/stapler/stapler/releases/tag/stapler-parent-1.260
+        title: Stapler 1.260 changelog
+    message: |-
+      Correctly encode control and surrogate characters when exporting data to JSON.
+  - type: bug
+    category: bug
+    pull: 4801
+    issue: 62714
+    authors:
+    - clywyk
+    message: |-
+      Replace esclave with agent in French documentation.
+  - type: bug
+    category: localization
+    pull: 4809
+    authors:
+    - stikonas
+    message: |-
+      Fix a typo in Lithuanian translation.
+  - type: rfe
+    category: rfe
+    pull: 4814
+    authors:
+    - timja
+    message: |-
+      Remove white background of 'Keep this build forever' when using dark theme.
+  - type: rfe
+    category: rfe
+    pull: 4808
+    issue: 62698
+    authors:
+    - fqueiruga
+    message: |-
+      Restyle hyperlinks to make them look more modern and readable.
+  - type: rfe
+    category: rfe
+    pull: 4783
+    authors:
+    - takashiharano
+    message: |-
+      Update the Japanese messages for the scheduler on the setting page.
+  - type: rfe
+    category: developer
+    pull: 4731
+    issue: 62345
+    authors:
+    - stellargo
+    references:
+      - issue: 62345
+      - pull: 4731
+      - url: https://github.com/jenkinsci/jep/tree/master/jep/226
+        title: JEP-226
+      - url: https://github.com/jenkinsci/redis-fingerprint-storage-plugin
+        title: Reference Implementation
+    message: |-
+      Developer: Add experimental External Fingerprint Storage API.
```

Comment:
Any reason why the first changelog suggestion was skipped in https://github.com/jenkinsci/jenkins/pull/4731 ? There are not only new APIs, but changes and deprecations in existing APIs